packages/amplication-design-system/src/style/css-variables.scss

```diff
@@ -54,20 +54,30 @@
 
   --form-horizontal-width: 700px;
   --form-horizontal-label-width: 300px;
-  --title1-font-size: 15px;
+  --large-icon-font-size: 18px;
+  --title1-font-size: 16px;
   --title2-font-size: 12px;
   --title3-font-size: 14px;
   --title4-font-size: 24px;
+  --icon-font-size: 16px;
   --normal-font-size: 12px;
   --text-field-label-font-size: 10px;
   --text-field-label-horizontal-font-size: var(--normal-font-size);
 
   --mdc-typography-body2-font-size: var(--normal-font-size);
 
+  --icon-size-xxsmall: 16px;
+  --icon-size-xsmall: 20px;
+  --icon-size-small: 24px;
+  --icon-size-medium: 32px;
+  --icon-size-large: 44px;
+  --icon-size-xlarge: 54px;
+  --icon-size-xxlarge: 64px;
+
   --icon-spacing: 4px;
   --default-spacing: 12px;
   --default-spacing-small: 8px;
-  --large-spacing: 16px;
+  --large-spacing: 14px;
```

Comment:
did you check where else we use this var and what is the impact?

Reply:
I didn't check but I asked @alexbass86 about it

Reply:
68 places.. you may be fixing one place to be exactly 14 as you want, but changing many other places that Alex may want to keep it 16... anyway, since we are working with multiple of 4, I don't think we should use 14 

<img width="426" alt="image" src="https://user-images.githubusercontent.com/43705455/184441532-35765781-9e1e-48ca-be6a-489f28476a9b.png">